lighthouse-core/report/html/report-styles.css

```diff
@@ -1410,6 +1411,9 @@
   width: var(--image-preview-size);
   height: var(--image-preview-size);
   display: block;
+  position: absolute;
```

Comment:
could you try using flexbox instead?

https://css-tricks.com/snippets/css/a-guide-to-flexbox/

should be something like set `display: flex; align-items: center` on the url column

Reply:
yeah, using `display: flex` was my original approach but I must have missed something because it wasn't working, I'll fix it!

Reply:
update @connorjclark so when I tried to use `display: flex; align-items: center;` on `.lh-table-column--url` or `.lh-table .lh-table-column--thumbnail` I wasn't getting any movement, I couldn't find the official docs for it but [this](https://stackoverflow.com/questions/48516538/flex-align-items-center-not-working/48516758) post suggests that using `align-items` only works if we explicitly sized the height of the container which I don't think we do

When I tried to check if explicitly sizing a column entry helped, it did center the child element within the parent container, but then the row was not evenly centered, as some elements had different column heights than others

The other method I tried out did not use any flexboxes at all and only involved overwriting `.lh-table tr`'s `vertical-align: baseline` to be `vertical-align: middle` in `.lh-table .lh-table-column--thumbnail`. This generally worked well:
<img width="881" alt="Screen Shot 2020-08-04 at 5 21 46 PM" src="https://user-images.githubusercontent.com/32499374/89358021-09ace700-d677-11ea-8465-1bf79c79bd0e.png">

Is the `vertical-align` method fine or should I continue looking at flexbox methods?

Reply:
>  this post suggests that using align-items only works if we explicitly sized the height of the container which I don't think we do

that doesn't seem right. for example, `lh-load-opportunity__col` is a flex container using align-items w/o an explicit height, and it works.

I forgot that flex doesn't work to well within table elements. https://stackoverflow.com/a/41421700/2788187 . 

given all this... I suppose the original PR wfm

Reply:
I don't pretend to know CSS too well, but I just nerdsniped myself and am going to read through https://www.w3.org/Style/CSS/read.en.html over the next couple weeks ..